go.mod

```diff
@@ -268,3 +268,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	lukechampine.com/blake3 v1.4.1 // indirect
 )
+
+// TODO: remove this once the new version is released. see https://github.com/ipfs/go-log/pull/160
+replace github.com/ipfs/go-log/v2 => github.com/sgtpooki/go-log/v2 v2.0.0-20250724190757-1d9b5d3dc8e2
```

Comment:
FYI I ran into some issues with updating this to the latest commit on the go-log PR so I am pointing it to my new fork

Reply:
Do you remember what are those issues? IF useful, the way I usually do PRs like this is:
- open PR draft with go-log changes
- in kubo do `go get github.com/ipfs/go-log/v2@{commit-from-pr} && make mod_tidy` -- this way you point at the right commit from the original repo

Pushed a change that did just that -- no need for fork.

Reply:
the errors were something about not being able to download the version.. or that it didn't exist. IIRC think they were 404s.. but I will use this pattern going forward. Thanks!